jinja2/utils.py

```diff
@@ -456,6 +457,13 @@ def values(self):
         return [x[1] for x in self.items()]
 
     def itervalue(self):
+        """Iterate over all values."""
+        warnings.warn(DeprecationWarning(
+            '"itervalue()" is deprecated, use "itervalues()" instead.'
```

Comment:
This should mention that it will be removed in version 2.12. Check out Werkzeug for an example of how I word deprecation warnings, this is pretty close already.